src/components/SimplerSelect.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { Select, SelectOptionObject, SelectProps } from '@patternfly/react-core';
+
+type Props = Omit<Omit<Omit<SelectProps, 'isExpanded'>, 'onSelect'>, 'onToggle'> & {
+  onSelect?: (selection: string | SelectOptionObject) => void;
+  onToggle?: (isExpanded: boolean) => void;
+};
+
+const SimplerSelect = (props: Props) => {
+  const [isExpanded, setExpanded] = React.useState(false);
```

Comment:
Oh! you are trying react hooks :)

Reply:
yes it's actually the second time, it's a lot less boilerplate when the situation allows it!